### ISSUE TITLE
Urgent. Some app developer is using iceraven browser icon. 

### ISSUE THREAD

**Issue by ghost** [fork-maintainers/iceraven-browser#729]
Urgent. Some app developer is using iceraven browser icon. 

Is this approved or something is misusing?

![Screenshot_2024-01-14-21-07-30-630_com android vending](https://github.com/fork-maintainers/iceraven-browser/assets/156216269/916c84df-9d65-48bc-9127-d704688161ec)

**Comment by akliuxingyuan** [comment]
> Is this approved or something is misusing?

No, absolutely not got approved, and interfect certainly didn't authorize it.

They stole Iceraven project's logo, I think that it violates [Google Play Developer Policy](https://play.google.com/about/developer-content-policy/) and project's Licence.

I submitted a form to report the app, @Varniit-pro-90s thank you for sharing this infomation.

**Comment by ghost** [comment]
> > Is this approved or something is misusing?
> 
> No, absolutely not got approved, and interfect certainly didn't authorize it.
> 
> They stole Iceraven project's logo, I think that it violates [Google Play Developer Policy](https://play.google.com/about/developer-content-policy/) and project's Licence.
> 
> I submitted a form to report the app, @Varniit-pro-90s thank you for sharing this infomation.

The same dev has copied apps and icons and other factors from other apps makers. And see pricing of apps that are generally free, misusing Google Play for bad practices.

And if he had put that up there, bring iceraven browser to Google Play for others.

**Comment by akliuxingyuan** [comment]
The app had been taken down, same as the author account, not sure about the details behind.
